### ISSUE TITLE
ECS Agent with Docker 1.9.1 talking to 1.7.1 endpoint?

### ISSUE THREAD

**Issue by pixeleet** [aws/amazon-ecs-agent#421]
Hey Folks,

Isn't that strange? 

```
time="2016-06-09T10:17:24.252079672Z" level=error msg="HTTP Error" err="no such id: 5b6624716d40cf3c3d29af8db548c330a777b1b0e7f4d40325ad7cf4a351434f" statusCode=404
time="2016-06-09T10:17:24.489150878Z" level=info msg="GET /v1.17/containers/9b569cca1ff4550995cf6ff2a144ac28bca1ef9695cc2b35ff2f833da941a71a/json"
time="2016-06-09T10:17:24.489552262Z" level=info msg="GET /v1.17/containers/9b569cca1ff4550995cf6ff2a144ac28bca1ef9695cc2b35ff2f833da941a71a/json"
time="2016-06-09T10:17:27.980825464Z" level=info msg="GET /containers/json?all=0&filters={%22status%22:%5b%22running%22%5d}"
time="2016-06-09T10:17:28.088577055Z" level=info msg="DELETE /v1.17/containers/6eddd171514aec5d3d1560ee54e694400b3f72bedeb20845e573e235f6089952?v=1"
time="2016-06-09T10:17:35.094841936Z" level=info msg="GET /containers/json?all=1&filters={%22status%22:%5b%22restarting%22%5d}&limit=100"
time="2016-06-09T10:17:38.105639580Z" level=info msg="GET /containers/json?all=1&filters={%22status%22:%5b%22paused%22%5d}&limit=100"
[ec2-user@ip-10-0-200-11 ~]$ docker -v
Docker version 1.9.1, build a34a1d5/1.9.1
```

**Comment by samuelkarp** [comment]
Hi @pixeleet,

I'm a bit confused by your question.  Can you clarify exactly what you're asking?

Taking a guess: I think you're looking at the API version 1.17 that you see in the Docker daemon log.  This is a versioned call to the [Docker Remote API](https://docs.docker.com/engine/reference/api/docker_remote_api/).  We [default](https://github.com/aws/amazon-ecs-agent/blob/v1.10.0/agent/engine/dockerclient/versions.go#L33) to 1.17 as that's the latest API version available in Docker 1.5.0 (which is our minimum supported version of Docker).  API version 1.17 is currently supported across all versions of Docker greater than 1.5.0.

Sam

**Comment by pixeleet** [comment]
My bad, I must have misread it while insufficient caffeine was in my system.

Closed.
